pkg/server/server.go

```diff
@@ -0,0 +1,74 @@
+package server
+
+import (
+	"fmt"
+	"github.com/labstack/echo/v4/middleware"
+	"net/http"
+	"time"
+
+	"github.com/labstack/echo/v4"
+)
+
+const basePhotosApiPath = "/v1/photos"
+
+type Server struct {
+	port       string
+	echoServer *echo.Echo
+}
+
+type status struct {
+	Status int `json:"status"`
+}
+
+func InitServer(port string) *Server {
+	// Echo instance
+	e := echo.New()
+	// Middleware
+	e.Use(middleware.Logger())
+
+	s := &Server{
+		port:       port,
+		echoServer: e,
+	}
+	s.AddEndpoints()
```

Comment:
Moved the `AddEndpoints()` call inside here so we don't have to keep calling it from the client code.  I figureed we would never want to start the service without the endpoints so might save a few lines by calling the method here.